<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QMainWindow" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="emailBox">
    <property name="geometry">
     <rect>
      <x>287</x>
      <y>211</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="passBox">
    <property name="geometry">
     <rect>
      <x>287</x>
      <y>271</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="passLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font-weight:bold;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="emailLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font-weight:bold;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cPassLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font-weight:bold;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="userNameBox">
    <property name="geometry">
     <rect>
      <x>287</x>
      <y>151</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="cPassBox">
    <property name="geometry">
     <rect>
      <x>287</x>
      <y>331</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="placeholderText">
     <string>Confirm Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signUpButton">
    <property name="geometry">
     <rect>
      <x>287</x>
      <y>391</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font-weight:bold;
background-color: rgb(8, 8, 8);</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QLabel" name="userLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);
font-weight:bold;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
